ReadMe.txt
copy both folder


Pro Leads/CHANGELOG.md
3.0.12 / 2017-09-12
==================

 * Fixed: some payment methods are not supported in older systems
 * Fixed: verification with thechecker.co sometimes does not work
 * Fixed: certain custom tags are not translated
 * Changed: do not show up raw bounce message

3.0.11 / 2017-8-31
==================

 * Fixed: suppress PHP warning messages from the UI
 * Fixed: intermittent issue with automation trigger 
 * Fixed: subscriber tags not translated
 * Added: support Paddle payment gateway

3.0.10 / 2017-08-22
==================

 * Fixed: payment issue with Stripe/Braintree
 * Fixed: tax billing information is configurable, no longer compulsory
 * Fixed: license verification does not work in certain cases
 * Fixed: speed up campaigns listing
 * Fixed: speed up subscribers counting
 * Fixed: automation trigger for date in the past
 * Fixed: reduce the number of SQL queries
 * Fixed: retain custom translation when upgrading
 * Added: allow resending a campaign
 * Added: support SendGrid subuser
 * Added: support sending a campaign again
 * Added: support verify-email.org service
 * Added: support importing subscribers via CLI

3.0.9 / 2017-07-13
==================

 * Fixed: bounced emails not added to blacklist
 * Fixed: Reply-To header not correctly set for SendGrid and ElasticEmail
 * Fixed: support Elastic {unsubscribe} tag
 * Added: include Portuguese translation
 * Added: speed up list loading
 * Added: support testing bounce/feedback handlers
 * Added: support sending a test email for sending server
 * Added: display the generated DKIM DNS record for sending domain
 * Added: reduce the tracking log size for faster delivery
 * Added: allow accessing the webapp from root directory
 * Added: support copying templates

3.0.8 / 2017-06-29
==================

 * Fixed: bounced email not added to blacklist
 * Fixed: make sure confirmation email be triggered
 * Added: reduce memory usage while sending
 * Added: speed up page load in general
 * Added: font selection support for email template
 * Added: speed up automation page load
 * Added: speed up subscribers listing

3.0.7 / 2017-06-19
==================

 * Fixed: message-id cannot be retrieved in bounced message
 * Fixed: JS error showing up in the edit template page
 * Fixed: lagging image/video in the email editor
 * Fixed: thumbnail not showing up correctly
 * Fixed: user cannot subscribe to certain plans
 * Fixed: speed up the customers list view

3.0.6 / 2017-06-08
==================

 * Fixed: issue with automation for subscriber events
 * Fixed: automation follow-up email not triggered in certain cases
 * Fixed: incorrect quota counting in certain cases
 * Fixed: cannot subscribe customer to a plan as administrator
 * Fixed: prevent double subscription
 * Fixed: check to make sure public/ folder is writable while installing
 * Changed: queued campaign is now of QUEUED status (READY is no longer used)
 * Changed: SMTP encryption is no longer a required field
 * Changed: remove GMP extension validation as it is no longer required

3.0.4 / 2017-06-01
==================

 * Added: subscribers count by unique email
 * Added: more test scripts
 * Fixed: fix potential security issues related to CSRF
 * Fixed: upgrade will no longer overwrite custom translation
 * Fixed: campaign does not pause in certain cases 
 * Fixed: CSV export may not work correctly

3.0.3 / 2017-05-25
==================

 * Added: support billing information page
 * Added: configure sending servers that can be added by customer
 * Added: look up GEO information from local database
 * Fixed: intermittent unsubscribe issue
 * Fixed: plain text glitch with ElasticEmail
 * Fixed: intermittent issue when saving payment methods
 * Fixed: update-profile URL does not work with automation
 * Fixed: editor inserts additional tags to the email content

3.0.2 / 2017-05-20
==================

 * Added: support working with Braintree's merchant accounts
 * Added: support updating site logo
 * Fixed: delivery handler issue with SSL
 * Fixed: eliminate duplicate background jobs
 * Fixed: automation issue with list subscription
 * Fixed: prevent duplicate unsubscription
 * Fixed: better mobile compatibility
 * Fixed: upgrade manager issue with old PHP versions

3.0.1 / 2017-05-11
==================

 * Added: integration with email verification service Kickbox.io
 * Added: integration with email verification service TheChecker.co
 * Added: upgrade manager - allow upgrading directly from the admin dashboard
 * Added: email web viewer (through the WEB_VIEW_URL tag)
 * Added: table features for email builder
 * Fixed: plain text campaign issue with ElasticEmail

3.0.0-p3 / 2017-04-20
==================

 * Added: payment integration with PayPal
 * Added: payment integration with Stripe
 * Added: preview campaign email before sending
 * Changed: change the API response format for CREATE USER, including api_token
 * Fixed: licence page error on certain systems
 * Fixed: duplicate queries generated for campaign statistics view

3.0.0-p1 / 2017-04-11
==================

 * Added: service plan management
 * Added: registration for visitor
 * Added: payment integration with Braintree (for Paypal and credit card)
 * Added: sending throttling setting
 * Added: subscription management
 * Added: customer management
 * Added: more flexible role based access control
 * Added: license verification
 * Added: clean up sending server's send() function
 * Added: more intuitive description on the UI
 * Fixed: automation sending duplicate follow-up email

2.2.0-p14 / 2017-03-30
==================

 * Added: support sending email through SparkPost API
 * Added: reduce memory usage by 40% for sending to 1M subscribers
 * Added: support file update via API
 * Added: better built-in  Spanish translation
 * Changed: suppress error messages in laravel.log
 * Fixed: import issue with non-break spaces
 * Fixed: check/uncheck sending server in Group Edit page
 * Fixed: feedback handling with ElasticEmail

Pro Builder/resources/views/users/users.blade.php
@extends('layouts.master')

@section('title')
Dashboard | Users
@endsection

@section('content')

@include('includes.nav-bar')

<div class="container-fluid">
	<div class="row">
		<div class="col-md-9 col-sm-8">
			<h1><span class="fui-user"></span> Users</h1>
		</div><!-- /.col -->

		<div class="col-md-3 col-sm-4 text-right">
			<a href="#newUserModal" data-toggle="modal" class="btn btn-lg btn-primary btn-embossed btn-wide margin-top-40"><span class="fui-plus"></span> Add New User</a>
		</div><!-- /.col -->
	</div><!-- /.row -->
	<hr class="dashed margin-bottom-30">
	<div class="row">
		<div class="col-md-12">
			@if( Session::has('success') )
			<div class="alert alert-success">
				<button type="button" class="close fui-cross" data-dismiss="alert"></button>
				{{ Session::get('success') }}
			</div>
			@endif
			@if( Session::has('error') )
			<div class="alert alert-error">
				<button type="button" class="close fui-cross" data-dismiss="alert"></button>
				{{ Session::get('error') }}
			</div>
			@endif
			<div class="masonry-4 users" id="users">
				@include('partials.users')
			</div><!-- /.masonry -->
		</div><!-- /.col -->
	</div><!-- /.row -->
</div><!-- /.container -->

<!-- Modals -->
@include('includes.modal-sitesettings')

@include('includes.modal-deletesite')

<div class="modal fade deleteUserModal" id="deleteUserModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
	<div class="modal-dialog">
		<div class="modal-content">
			<div class="modal-header">
				<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
				<h4 class="modal-title" id="myModalLabel"><span class="fui-info"></span> Are you sure?</h4>
			</div>
			<div class="modal-body">
				<p>Deleting this user account will result in all associated data being deleted (with the exception of externally published funnels) and <b>can not be undone</b>. Are you sure you want to continue?</p>
			</div><!-- /.modal-body -->
			<div class="modal-footer">
				<button type="button" class="btn btn-default" data-dismiss="modal"><span class="fui-cross"></span> Cancel</button>
				<a href="" type="button" class="btn btn-primary" id="deleteUserButton"><span class="fui-check"></span> Yes, I'm sure</a>
			</div>
		</div><!-- /.modal-content -->
	</div><!-- /.modal-dialog -->
</div><!-- /.modal -->

<div class="modal fade newUserModal" id="newUserModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
	<form class="form-horizontal" role="form" action="{{ route('user-create') }}">
		<div class="modal-dialog">
			<div class="modal-content">
				<div class="modal-header">
					<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
					<h4 class="modal-title" id="myModalLabel"><span class="fui-user"></span> Create a new user account</h4>
				</div>
				<div class="modal-body padding-top-40">
					<div class="loader" style="display: none;">
						<img src="{{ URL::to('src/images/loading.gif') }}" alt="Loading...">
						Creating new account...
					</div>
					<div class="modal-alerts"></div>
					<div class="form-group">
						<label for="username" class="col-md-3 control-label">First name:</label>
						<div class="col-md-9">
							<input type="text" class="form-control" id="first_name" name="first_name" placeholder="First name" value="">
						</div>
					</div>
					<div class="form-group">
						<label for="username" class="col-md-3 control-label">Last name:</label>
						<div class="col-md-9">
							<input type="text" class="form-control" id="last_name" name="last_name" placeholder="Last name" value="">
						</div>
					</div>
					<div class="form-group">
						<label for="username" class="col-md-3 control-label">Email:</label>
						<div class="col-md-9">
							<input type="email" class="form-control" id="email" name="email" placeholder="Email" value="">
						</div>
					</div>
					<div class="form-group">
						<label for="password" class="col-md-3 control-label">Password:</label>
						<div class="col-md-9">
							<input type="password" class="form-control" id="password" name="password" placeholder="Password" value="">
						</div>
					</div>
					<div class="form-group">
						<div class="col-md-offset-3 col-md-9">
							<label class="checkbox" for="type" style="padding-top: 0px;">
								<input type="checkbox" value="admin" name="type" id="type" data-toggle="checkbox">
								Admin permissions
							</label>
						</div>
					</div>
				</div><!-- /.modal-body -->
				<input type="hidden" name="_token" value="{{ Session::token() }}">
				<div class="modal-footer">
					<button type="button" class="btn btn-default" data-dismiss="modal"><span class="fui-cross"></span> Cancel</button>
					<button type="button" class="btn btn-primary" id="buttonCreateAccount"><span class="fui-check"></span> Create Account</button>
				</div>
			</div><!-- /.modal-content -->
		</div><!-- /.modal-dialog -->
	</form>
</div><!-- /.modal -->

@include('includes.modal-account')

<!-- /Modals -->

<!-- Load JS here for greater good =============================-->
<script src="{{ URL::to('src/js/vendor/jquery.min.js') }}"></script>
<script src="{{ URL::to('src/js/vendor/flat-ui-pro.min.js') }}"></script>
<script src="{{ URL::to('src/js/vendor/jquery.zoomer.js') }}"></script>
<script src="{{ URL::to('src/js/build/users.js') }}"></script>
<!--[if lt IE 10]>
<script>
$(function(){
	var msnry = new Masonry( '#users', {
    	// options
    	itemSelector: '.user',
    	"gutter": 20
    });

})
</script>
<![endif]-->

@endsection
